spring-cloud-gcp-autoconfigure/src/main/java/com/google/cloud/spring/autoconfigure/secretmanager/GcpSecretManagerProperties.java

```diff
@@ -43,6 +44,13 @@ public class GcpSecretManagerProperties implements CredentialsSupplier {
    */
   private String projectId;
 
+  /**
+   * Defines the region of the secrets when Regional Stack is used.
+   *
+   * <p>When not specified, the secret manager will use the Global Stack.
```

Comment:
Is it syntactically correct if there is no closing `</p>` tag?

```suggestion
   * <p>When not specified, the secret manager will use the Global Stack.</p>
```